### DISCUSSION TITLE
内置aria2离线下载，能不能加上改名功能

### DISCUSSION THREAD

**Discussion by hbqjren** [AlistGo/alist#6702] [Ideas]
我常用移动云盘，经常离线下载文件。它下载后，移动云盘官方默认文件后缀名改不了，有点苦恼。我就想在离线下载介面。一个“改名”点钩选项，下载后改名上传
![Screenshot_2024-07-04-23-29-16-718_mark via](https://github.com/alist-org/alist/assets/82219093/9014c3e8-c60d-4b4b-8320-0f0194e221a9)

**Comment by orion-tan** [comment]
复议，多行链接可以在每个链接后面用字符分割后加上重命名文件名称，没有分割就不用重命名。
